HaulageApplication/Haulage/BaseClasses/TripHandler/Item.cs

```diff
@@ -6,13 +6,32 @@
 
 namespace Haulage.BaseClasses.TripHandler
 {
+    
     public class Item
     {
+        public enum ItemCategoryType
+        {
+            Fragile,
```

Comment:
instead of having an enum here would it be worth have two true/false fields which called IsFragile and IsDangerous. This would allow for the scenario where an item is fragile and Dangerous. May not have time to implement this change however due to time constraints

Reply:
Yes, due to time constraints, and the requirements not specifying both could occur at the same time, I will leave it for now, and potentially add as an issue for future improvements?